__tests__/app.test.js

```diff
@@ -119,6 +117,26 @@ describe("app()", () => {
             );
           });
       });
+      it("200: return with a status of 200 and the corresponding article 5 data, including the number of comments that article may have", () => {
+        return request(app)
+          .get("/api/articles/5")
+          .expect(200)
+          .then(({ body }) => {
+            const { article } = body;
+            expect(article).toHaveProperty("article_id", 5);
+            expect(article).toHaveProperty("comment_count", "2");
+            expect(article).toHaveProperty("author", expect.any(String));
+            expect(article).toHaveProperty("title", expect.any(String));
+            expect(article).toHaveProperty("body", expect.any(String));
+            expect(article).toHaveProperty("topic", expect.any(String));
+            expect(article).toHaveProperty("created_at", expect.any(String));
+            expect(article).toHaveProperty("votes", expect.any(Number));
+            expect(article).toHaveProperty(
+              "article_img_url",
+              expect.any(String)
```

Comment:
There are a lot of repeated lines in this test that are the same as the test before it. If this test is only interested in the `comment_count` property then that's the only property you need to test for here 👍